model-optimizer/extensions/ops/TFResizeBilinear.py

```diff
@@ -0,0 +1,73 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from mo.front.common.layout import get_height_dim, get_width_dim
+from mo.front.common.partial_infer.utils import int64_array
+from mo.graph.graph import Node, Graph
+from mo.ops.op import Op
+
+
+class TFResizeBilinearOp(Op):
+    op = 'TFResizeBilinear'
+
+    def __init__(self, graph: Graph, attrs: dict):
+        mandatory_props = {
+            'op': self.op,
+            'out_ports_count': 1,
+            'in_ports_count': 2,
+            'infer': TFResizeBilinearOp.tf_resize_bilinear_infer
+        }
+        super().__init__(graph, mandatory_props, attrs)
+
+    def supported_attrs(self):
+        return [
+            'align_corners',
+            'half_pixel_centers'
+        ]
```

Comment:
Not needed since the operation is not emitted to IR.

Reply:
Done.